packages/tools/client-debugger/client-debugger-chrome-extension/webpack.config.js

```diff
@@ -59,7 +59,13 @@ module.exports = {
 			process: "process/browser",
 		}),
 		new CopyPlugin({
-			patterns: [{ from: ".", to: ".", context: "public" }],
+			patterns: [
+				// Copy assets from `public`
+				{ from: ".", to: ".", context: "public" },
+
+				// Copy HTML resources from source
+				{ from: "**/*.html", to: ".", context: "src" },
+			],
```

Comment:
Hmm the previous PR didn't introduce new HTML files... was this broken since before that?

Reply:
Prior to that PR, our standard build step was copying them (which it still does), but now we have a separate build directory for the webpack bundle output, and they weren't being copied there.